abstio/src/lib.rs

```diff
@@ -32,5 +32,5 @@ mod io;
 /// and reading files in different environments.
 pub fn slurp_bytes(filename: &str) -> Vec<u8> {
     let path = path(filename);
-    slurp_file(&path).expect(&format!("Can't read {}", path))
```

Comment:
Yeah, this is a good lint. It would be great to have a more ergonomic variant for this, however it might be better to refactor the code to use `?` instead.

Reply:
Making most things in `abstio` return `Result` is on my list. I'll try and cover that in an upcoming refactor to this crate (to properly have a `Filesystem` abstraction representing the local filesystem on native, remote files available according to a manifest on native and web, and a merged view for native). In the meantime, the change here is fine